### ISSUE TITLE
What are the legal implications and obligations of 'contract'

### ISSUE THREAD

**Issue by Wiladams** [deptofdefense/code.mil#48]
Since I am new to seeing a 'contract' instead of a simply copyright license, I'm not sure of the implications of such.  What if any obligations do I incur because of this?  What liability am I exposed to.

It's a bit wordy, so I'm just not sure how it stands relative to some thing like the WTFPL.

Looking for some clarification on those unknown implications.  And looking for a simpler contract/license.

**Comment by johnmod3** [comment]
1. Maybe instead of trying to enforce copyright law via contract law, focus on a statement of government intent or desire for projects to evolve into fully realized and managed open source software projects. 
Something like: release as public domain with a statement that the government will only accept code submissions back under an OSS copyright. That way over time the codebase slowly gets a copyright on it. 

2. Or have a contractor involved in each project so they can assert copyright. 

3. Longer term fix I think is to push Congress to allow the DoD/IC to assert copyright and then dictate it be released (its the only way to be certain)

**Comment by shawoods** [comment]
@johnmod3 Using contract law really was an experiment to see how it might be received in the community. The priority is achieving fully realized open source projects. We are pivoting toward something like what you suggest in #1.  We have also discussed #3, but that's definitely a long game play.

**Comment by BrandonBouier** [comment]
We've pivoted based on user feedback. Take a look at the current CONTRIBUTING.md

**Comment by fulldecent** [comment]
The copyright aspect of this project has been removed from the scope of this project based on the updated CONTRIBUTING file. Therefore I believe this issue is now out of scope and should be closed.

**Comment by BrandonBouier** [comment]
Got it. Thanks @fulldecent!
